lua/neotest-elixir/core.lua

```diff
@@ -0,0 +1,214 @@
+local lib = require("neotest.lib")
+local Path = require("plenary.path")
+local toggleterm = require("toggleterm")
```

Comment:
I don't love the dependency on `toggleterm` here; especially when it's not called out as a dependency for this plugin in the README, etc. I use my own `term` module, and not `toggleterm` (granted, toggleterm.nvim is the gold-standard for term plugins at the moment). So this results in a runtime exception that toggleterm can't be found. Could we add a `pcall` here to catch and handle that?

Reply:
Makes sense,  for someone who doesn't test in the IEx shell, this dependency is indeed unnecessary.

Reply:
I agree. It should be an optional dependency.